common/azure/build.gradle.kts

```diff
@@ -11,7 +11,8 @@ dependencies {
 
   api("com.azure.spring:azure-spring-boot-starter-cosmos")
   implementation("com.azure.spring:azure-spring-boot-starter-storage")
-  api("com.azure:azure-storage-blob-batch")
+  implementation("com.azure:azure-storage-blob-batch:12.11.3")
+  api("com.azure:azure-storage-blob-batch:12.11.3")
```

Comment:
The goal of the BOM is to make sure we won't have potential dependency conflicts with the versions that transitively come from the `com.azure` group, as was the case recently.
I think it would make more sense to understand what could have changed in the parent BOM dependencies, and to see whether this is intentional or not.

Reply:
https://github.com/microsoft/azure-spring-boot/issues/949

Reply:
As commented out in https://github.com/microsoft/azure-spring-boot/issues/949#issuecomment-1031139920 and https://github.com/Azure/azure-sdk-for-java/issues/26136#issuecomment-1005955468, the Azure SDK BOM is purposely no longer a dependency of the Azure Spring Boot BOM.
So we now need to additionally import the Azure SDK BOM and drop the explicit version for `azure-storage-blob-batch`.